src/Controller/OgAdminMembersController.php

```diff
@@ -21,13 +53,117 @@ class OgAdminMembersController extends ControllerBase {
    *   The members overview View.
    */
   public function membersList(RouteMatchInterface $route_match) {
-    $parameter_name = $route_match->getRouteObject()->getOption('_og_entity_type_id');
-
     /** @var \Drupal\Core\Entity\EntityInterface $group */
-    $group = $route_match->getParameter($parameter_name);
+    $group = $this->getGroupFromRoute($route_match);
 
     $arguments = [$group->getEntityTypeId(), $group->id()];
     return Views::getView('og_members_overview')->executeDisplay('default', $arguments);
   }
 
+  /**
+   * Displays add membership links for available membership types.
+   *
+   * Returns default membership type if that's all that exists.
+   *
+   * @return array
+   *   A render array for a list of the node types that can be added; however,
```

Comment:
@jludwig 
In the function, it seems there's nothing about nodes. but about og_memberships. `list of the node types` and later `is only one node type` should be in-line with the actual entity that we deal with.